polyfill/lib/ecmascript.mjs

```diff
@@ -791,18 +791,18 @@ export const ES = ObjectAssign({}, ES2019, {
       return null;
     }
 
+    const current = new TemporalAbsolute(ES.SystemUTCEpochNanoSeconds());
```

Comment:
I don’t think we need to construct an extra temporal object here. Do we?

This entire function works off raw bigint epochNanoSeconds

Reply:
I literally just construct this function to use the fancy `Absolute` math (see! It works!)

I didn't think it would be a big deal, but if you feel strongly about it, I'd do the nanosecond math manually.

Reply:
I don’t feel strongly. Just thinking it adds unnecessary complexity 😀 but it really doesn’t matter much.

Reply:
@pipobscure what about now? :smile: